envs/.env.template

```diff
@@ -0,0 +1,11 @@
+MINER_HOTKEY=
+AWS_MINER_INSTANCE_ID=
+MINER_INSTANCE_PORT=
+
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_REGION_NAME=eu-north-1
+AWS_S3_BUCKET_NAME=
+AWS_ROUTE53_HOSTED_ZONE_ID=
+
+SQL_ALCHEMY_DB_URL=sqlite:///db/ddos_shield.db
```

Comment:
i don't think this should be mentioned as an option to users, better to have the Dockerfile do `ENV SQL_ALCHEMY_DB_URL=sqlite:///db/ddos_shield.db`